db/seeds.rb

```diff
@@ -7,13 +7,14 @@
 end
 providence.save!
 
-Location.find_or_initialize_by(
+boulder = Location.find_or_initialize_by(
```

Comment:
Does this initialize to a boulder variable? In general, should this be done for the other towns?

Reply:
The variable is only referenced by initialized "TrafficCam"  objects - just as a means of differentiating between which should be displayed to the widget.